content/applications/sales/crm/acquire_leads/generate_from_website.rst

```diff
@@ -1,78 +1,99 @@
-===========================================================
-Generate leads/opportunities from your website contact page
-===========================================================
+============================
+Generate leads/opportunities
+============================
 
-Automating the lead/opportunity generation will considerably improve
-your efficiency. Any visitor using the contact form on your website will
-create a lead/opportunity in the pipeline.
+Automating lead/opportunity generation can significantly improve your team's efficiency. In this
+article, learn how to use email aliases and website contact forms to automatically generate leads
+in your database.
```

Comment:
Intros rather explain what it is about (even if it seems obvious to the writer) rather than promote the feature or list the content of the document.
https://www.odoo.com/documentation/master/contributing/documentation/content_guidelines.html#writing-style

Reply:
@meng-odoo let's circle back to Jonathan's comment here after I fix redirects file + the beefy screenshots on next commit 👍